entitycore_svc/variables.tf

```diff
@@ -0,0 +1,52 @@
+variable "aws_region" {
+  type = string
+}
+
+variable "vpc_id" {
+  type = string
+}
+
+variable "allowed_source_ip_cidr_blocks" {
+  type = list(string)
+}
+
+variable "private_alb_listener_arn" {
+  type = string
+}
+
+variable "root_path" {
+  description = "Base path for the API"
+  type        = string
+}
+
+variable "keycloak_url" {
+  description = "Keycloak URL"
+  type        = string
+  default     = "https://openbluebrain.com/auth/realms/SBO/"
```

Comment:
I'd personaly remove the default values. IMO it is better to fail because we are not setting a variable than deploying it with the wrong default values.